### PR TITLE
Prepare modelgauge infra files for safety eval test

### PR DESCRIPTION
**Pull request by tsunamit**
In preparation for the safety eval test plugin code, needed to make a few changes to the existing infra.
1. Enable system chat role
2. Add mistral 8x22b, llama 3 70b to together client
3. Small refactor of `llama_guard_annotator` to expose llama guard response translation method for reusability

### CODE REVIEW

File: modelgauge/prompt.py [hunk 1]
```diff
@@ -47,6 +47,7 @@ class SUTOptions(BaseModel):
 class ChatRole(str, Enum):
     user = "USER"
     sut = "SUT"
+    system = "SYSTEM"
```
Comment (tsunamit) [inline modelgauge/prompt.py:50]: (1) Add system role. Need this so we can specify system prompts in SUTs

File: plugins/together/modelgauge/annotators/llama_guard_annotator.py [hunk 2]
```diff
@@ -264,18 +264,33 @@ def translate_response(
...
-            return LlamaGuardAnnotation(is_safe=False, violation_categories=categories)
-        else:
-            raise AssertionError("Unexpected response: ", response.choices[0].text)
+        response_str = response.choices[0].text
+        return translate_llama_guard_response(
+            response=response_str, version=self.llama_guard_data, decoder=self.decoder
+        )
+
+
+def translate_llama_guard_response(
```
Comment (tsunamit) [inline plugins/together/modelgauge/annotators/llama_guard_annotator.py:273]: (3) Refactor to expose the llama guard response translation to make it a reusable method. We wanted to reuse this logic in our test

File: plugins/together/modelgauge/suts/together_client.py [hunk 3]
```diff
@@ -39,6 +39,7 @@ def description(cls) -> SecretDescription:
 _ROLE_MAP = {
     ChatRole.user: _USER_ROLE,
     ChatRole.sut: _ASSISTANT_ROLE,
+    ChatRole.system: _SYSTEM_ROLE,
```
Comment (tsunamit) [inline plugins/together/modelgauge/suts/together_client.py:42]: (1) add system role to together client to allow for system prompts

File: plugins/together/modelgauge/suts/together_client.py [hunk 4]
```diff
@@ -447,13 +448,15 @@ def translate_response(
     "llama-2-7b-chat": "meta-llama/Llama-2-7b-chat-hf",
     "llama-2-13b-chat": "meta-llama/Llama-2-13b-chat-hf",
     "llama-2-70b-chat": "meta-llama/Llama-2-70b-chat-hf",
+    "llama-3-70b-chat": "meta-llama/Llama-3-70b-chat-hf",
```
Comment (tsunamit) [inline plugins/together/modelgauge/suts/together_client.py:451]: (2) add llama-3-70b-chat and mistral-8x22b-instruct

File: plugins/together/modelgauge/annotators/llama_guard_annotator.py [hunk 5]
```diff
@@ -264,18 +264,33 @@ def translate_response(
...
+    response: str, version: LlamaGuardVersion, decoder: Optional[Dict[str, str]] = None
```
Comment (tsunamit) [inline plugins/together/modelgauge/annotators/llama_guard_annotator.py:274]: TODO fixing this rn

**Review by dhosterman — APPROVED** [review]
Looks great, thanks for the contribution!

**Pull request merged**